### PR TITLE
adding plugin to flip tables

### PR DESCRIPTION
**Pull request by juzten**

### CODE REVIEW

File: plugins/tableflip.ex [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  @usage """
+  butler tableflip - replies with flipped tables.
+  """
+  respond ~r/tableflip/, conn do
+    reply conn, "(╯°□°）╯︵ ┻━┻"
+    reply conn, "(╯°□°）╯︵ <ǝlqɐʇ>"
+    reply conn, "the table flipped you! ノ┬─┬ノ ︵ ( \o°o)\ "
+    reply conn, "┻━┻ ︵ヽ(`Д´)ﾉ︵ ┻━┻"
+  end
```
Comment (keathley) [inline plugins/tableflip.ex:26]: Do you want to reply with all of these or just 1 of them?

Comment (juzten) [inline plugins/tableflip.ex:26]: Yep, I'll update this to do one random string shortly.

File: plugins/tableflip.ex [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  def flip_table do
+    :random.seed(:os.timestamp)
+    table_list = [
+        "(╯°□°）╯︵ ┻━┻",
+        "(╯°□°）╯︵ <ǝlqɐʇ>",
+        "the table flipped you! ノ┬─┬ノ ︵ ( \o°o)\ ",
+        "┻━┻ ︵ヽ(`Д´)ﾉ︵ ┻━┻"
+      ]
+
+    hd(Enum.take_random(table_list, 1))
```
Comment (keathley) [inline plugins/tableflip.ex]: You can just use `Enum.random(table_list)` instead of having to explicitly use `hd`

**Comment by juzten** [comment]
@keathley I updated to use Enum.random

**Comment by keathley** [comment]
LGTM

**Pull request merged**